### ISSUE TITLE
Unable to login with Emby Connect email address

### ISSUE THREAD

**Issue by sgtcoder** [Fallenbagel/jellyseerr#749]
### Description

It would be nice to be able to login with the Emby Connect email address vs having to login with your first name and last name. Seems a bit silly

### Desired Behavior

Allow login with the Emby Connect email address rather than your first name and last name.

### Code of Conduct

- [X] I agree to follow Jellyseerr's Code of Conduct

**Comment by Fallenbagel** [comment]
We only use username/password logins

**Comment by sgtcoder** [comment]
You mean the first name and last name is the username.

**Comment by Fallenbagel** [comment]
> You mean the first name and last name is the username.

No. I mean we specifically use username. It's an emby issue that they set the first name/last name as username. We don't do that.

**Comment by sgtcoder** [comment]
Understandable, that's a bit odd on Emby's part.

Anyway we can use the email address as well? Everyone is a bit confused on the name for the username.

**Comment by Fallenbagel** [comment]
> Email logins via Emby connect users have been working for well over a year. All of a sudden they have stopped working and now users have to use their actual usernames now?
> 
> Did something change ?

Nothing changed. It was always username. we dont even provide support for emby connect. Never did. If it worked before, then its probably emby updating their api.

If you are still sure it used to work on jellyseerr and this is not from emby side, you could whip up a new container instance and use a previous version and check

**Comment by sgtcoder** [comment]
@Fallenbagel 

This is not true. I just did a POST login through /emby/Users/AuthenticateByName and it supports both name and email. The same request you make through Jellyserr as I saw in the logs.

```
Username
PW
```

**Comment by Fallenbagel** [comment]
> @Fallenbagel 
> 
> This is not true. I just did a POST login through /emby/Users/AuthenticateByName and it supports both name and email. The same request you make through Jellyserr as I saw in the logs.
> 
> ```
> Username
> PW
> ```

Nothing changed on our side is what I'm saying. We never used emails.

**Comment by sgtcoder** [comment]
Well the API call you use to connect to the Emby server, supports the name and username/email.

It is NOT Emby's issue. I literally just connected through their API and validated a response with email and/or NAME.

```
curl --location 'https://EMBY_SERVER/emby/Users/AuthenticateByName' \
--header 'Content-Type: application/json' \
--header 'X-Emby-Authorization: ••••••' \
--data-raw '{
    "Username":"EMBY_CONNECT_EMAIL",
    "PW":"PASSWORD_HERE"
}'
```

**Comment by Fallenbagel** [comment]
> Well the API call you use to connect to the Emby server, supports the name and username/email.
> 
> It is NOT Emby's issue. I literally just connected through their API and validated a response with email and/or NAME.
> 
> ```
> curl --location 'https://EMBY_SERVER/emby/Users/AuthenticateByName' \
> --header 'Content-Type: application/json' \
> --header 'X-Emby-Authorization: ••••••' \
> --data-raw '{
>     "Username":"EMBY_CONNECT_EMAIL",
>     "PW":"PASSWORD_HERE"
> }'
> ```

You're still not understanding what I'm saying. We never USED emails. We always USED usernames. It is not our fault that they CHOOSE the first name and last name as the username.

**Comment by Fallenbagel** [comment]
@sgtcoder I am moving the discussion here.

We weren't claiming its an emby issue. Youre still misunderstanding the entire thing.

We NEVER USED emails. We have always USED usernames. We cannot SUPPORT emails until we refactor jellyseerr.

Your original query was that it's silly that we were using first name last name as a username. Thats not something we decided. That's the username that emby connect sets up afaik when using emby connect. We never said that it was an "issue" that emby "doesn't" support. 

It was about your original query that we use first name last name as a username which was silly. 

You're not really understanding what I'm saying. I never claimed emby doesn't support email authentication. WE DONT. Because WE don't support emby per se either. The support is a happy accident through jellyfin api. Until we can refactor to decouple jellyfin and emby which is literally achieved by #685 that i linked, which you chose to ignore.

**Comment by Fallenbagel** [comment]
You're not really understanding what I'm saying. I never claimed emby doesn't support email authentication. WE DONT. Because WE don't support emby per se either. The support is a happy accident through jellyfin api. Until we can refactor to decouple jellyfin and emby which is literally achieved by #685 that i linked, which you chose to ignore.

**Comment by sgtcoder** [comment]
Yes, I understand that now. And you literally just linked it and I responded back to it: https://github.com/Fallenbagel/jellyseerr/pull/685#issuecomment-2133697403

Thank you for the clarification now. I think it could have been summarized as:

We currently don't support Emby Connect Email addresses, but we have a future update that should handle that. https://github.com/Fallenbagel/jellyseerr/pull/685

Rather than

> So it is EMBY'S issue that they set the first name last name as the username when connected through emby connect. Not > ours. All we do is authenticate using whatever username is available through emby

**Comment by Fallenbagel** [comment]
> Yes, I understand that now. And you literally just linked it and I responded back to it: https://github.com/Fallenbagel/jellyseerr/pull/685#issuecomment-2133697403
> 
> Thank you for the clarification now. I think it could have been summarized as:
> 
> We currently don't support Emby Connect Email addresses, but we have a future update that should handle that. https://github.com/Fallenbagel/jellyseerr/pull/685
> 
> Rather than
> 
> > So it is EMBY'S issue that they set the first name last name as the username when connected through emby connect. Not > ours. All we do is authenticate using whatever username is available through emby
> 
> 

Your first query was "it's silly that we use first name last name as the username". We don't. We use emby's username. Which is the first name last name when connected through emby connect. It's maybe silly but it's not something that we set.

**Comment by sgtcoder** [comment]
It is true that it is silly that the First Name and Last Name is used as the username. But after reviewing, that seems to come through the Emby side.

However, there is the Emby Connect Email address that would also work on AuthenticateByName API call. I realize now that there are currently some limitations on Jellyserr regarding that and there is an open PR that should hopefully address that.

Thank you for the hard work and the support for Emby.

Again, let me know in anyway I can be of help, whether it is testing or reviewing.
